packages/ui/ImageItem/ImageItem.module.less

```diff
@@ -4,26 +4,39 @@
 	box-sizing: border-box;
 
 	.image {
+		position: relative;
 		display: block; // leaving this to prevent whitespace between inline-block elements
 		margin: 0;
 	}
-	
+
 	.caption,
 	&.selected {
+		width: 100%;
 		/* Public Class Names */
 	}
-	
+
 	&.horizontal {
 		.image {
+			display: inline-block;
+			vertical-align: middle;
 			width: 25%;
 			height: 100%;
 		}
+		.image+div {
+			display: inline-block;
+			vertical-align: middle;
+			width: 75%;
+		}
 	}
-	
+
 	&.vertical {
+		height: 100%;
 		.image {
 			width: 100%;
```

Comment:
I couldn't understand why we need the width 100% here.
For `block`, it's the 100% by default. For `inline`, the width won't work.